project/SparkCassandraConnectorBuild.scala

```diff
@@ -59,14 +61,82 @@ object CassandraSparkBuild extends Build {
         "org.scala-lang" % "scala-compiler" % scalaVersion.value))
   ).disablePlugins(AssemblyPlugin, SparkPackagePlugin) configs IntegrationTest
 
-  lazy val connector = CrossScalaVersionsProject(
+  /**
+    * Do not included shaded dependencies so they will not be listed in the Pom created for this
+    * project.
```

Comment:
Yep Provided means that it won't be downloaded in the default configuration, try doing the publishM2 routine with spark-shell and you'll see it won't get grabbed.